### ISSUE TITLE
List item number not announcing on each list item and aria-describedby announce two times.

### ISSUE THREAD

**Issue by dalipkumar703** [FreedomScientific/VFO-standards-support#335]
### Summary
When there are listitems wrapped in a list, the screen reader doesn't announce "1 of x" for all list items, but only for the first list item of the list that the user navigates to - navigating to other list item within the list does not announce the index number of that list item. However, if the listitems are inside a simple div instead of a list, this issue does not occur.

Also, JAWS seems to announce the description for a listitem twice, regardless of whether the listitem is inside a list or a plain div.
###Example
> 1. Go to https://sarcasticneedyextraction--dalipkumar703.repl.co/
> 2. There two list and defined. First using role="list" which doesn't work correctly. Second, without role="list" that work properly.

#### JAWS version and build number
Jaws 2018.1811.30

 
#### Operating System and version
Window 10

#### Browser and version:
Firefox 69.0.3

**Comment by JAWS-test** [comment]
Checked your examples with JAWS 2019 and Firefox, Chrome and IE 11. Both problems do not occur any more

**Comment by dalipkumar703** [comment]
Thanks for update @JAWS-test .

**Comment by AMANDUA** [comment]
@JAWS-test - is there a way to upgrade to JAWS 2019 from JAWS 2018 or we have to explicitly buy JAWS 2019

**Comment by JAWS-test** [comment]
I am not from freedom scientific. Please ask the manufacturer of Jaws. As far as I know you can upgrade. Or you use the free 40 minutes demo version

**Comment by AMANDUA** [comment]
Okay...Thanks @JAWS-test

**Comment by JeremiahRogersVispero** [comment]
Because the [cited example](https://sarcasticneedyextraction--dalipkumar703.repl.co/) is no longer available and [issue comments](https://github.com/FreedomScientific/standards-support/issues/335#issuecomment-541084195) indicate the issue to have been resolved, closing the issue.
